PostProcessing/Runtime/Effects/ColorGrading.cs

```diff
@@ -224,7 +224,9 @@ void RenderHDRPipeline(PostProcessRenderContext context)
                         break;
                 }
 
-                int groupSize = Mathf.CeilToInt(k_Lut3DSize / 8f);
+				bool isMobilePlatform = Application.platform == RuntimePlatform.Android;
```

Comment:
`isMobilePlatform` should probably be called `isAndroidPlatform` if it only concerns Android.

Reply:
Also, please use spaces instead of tabs, following the internal Unity code conventions :)